packages/stream-object/src/index.ts

```diff
@@ -33,6 +64,36 @@ export const streamObject = async <T extends Schema>(options: StreamObjectOption
     schemaDescription: undefined,
     schemaName: undefined,
   }).then(({ chunkStream, stepStream, textStream: rawTextStream }) => {
+    let elementStream: ReadableStream<Infer<T>> | undefined
+    let index = 0
+
+    if (options.output === 'array') {
+      let textStream
+      ;[textStream, rawTextStream] = rawTextStream.tee()
```

Comment:
```suggestion
      let [textStream, rawTextStream] = rawTextStream.tee()
```

Reply:
will yield many errors
this is intended

Reply:
> will yield many errors this is intended

Okay, this looks like a naming error.